### PR TITLE
Add parallelization test

### PR DESCRIPTION
**Pull request by misialq**
This PR adds a Jupyter notebook containing parallelization analysis (3 iterations of sequence fetches for 20 run IDs, repeated for 1 and 4 jobs) and its results.

### CODE REVIEW

File: parallelization_test/parallelization-benchmark.ipynb [hunk 1]
```diff
@@ -0,0 +1,1325 @@
+{
```
Comment (adamovanja) [inline parallelization_test/parallelization-benchmark.ipynb:1]: <p>How about adding a brief title and description in a markdown cell above, sth like:</p><h1>Parallization analysis</h1><p>This notebook analyses the time required for running <code>get-sequences</code> with different number of jobs (1 and 4) in 3 iterations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (adamovanja) [inline parallelization_test/parallelization-benchmark.ipynb:1]: <p>the way it is set up now both fetches are run even when we are not interested in the processing time as <code>benchmark_result.csv</code> exists. How about moving the fetching data into the condition if the file does not exist already (below <code>if not os.path.isfile('benchmark_result.csv'):</code> ) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (adamovanja) [inline parallelization_test/parallelization-benchmark.ipynb:1]: <p>How about creating a quick function out of this code chunk as it's also repeated below?</p><pre class="ql-syntax" spellcheck="false">def calculate_time(n_jobs):
  []
  return times_many
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (adamovanja) [inline parallelization_test/parallelization-benchmark.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.        results_df = pd.DataFrame(
</pre><p>would add the fetching of both total times here (see comment above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (adamovanja) [inline parallelization_test/parallelization-benchmark.ipynb:1]: <p>could you add a brief comment in the Markdown cell where this data (benchmark_download_times.csv') is coming from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Review by adamovanja — CHANGES_REQUESTED** [review]
thanks @misialq a really nice visualization for our manuscript. I added some code comments inline.

**Comment by misialq** [comment]
Hey @adamovanja, thanks for your comments! I changed/added the small things according to your comments but I won't be doing any of the other code refactors, as it is highly unlikely that either me or anyone else will ever look at/re-run this notebook in the future...

**Review by adamovanja — APPROVED** [review]
alright, LGTM then.

**Pull request merged**